react-apps/self-care-portal/self-care-portal-frontend/accelerator/src/landing_page/Home.jsx

```diff
@@ -19,7 +19,7 @@
 import React, {useEffect, useContext} from "react";
 import {Nav, Footer} from "../common/index.js";
 import {Body} from "../landing_page";
-import {Switch, Route} from "react-router-dom";
+import {Switch, Route, useHistory} from "react-router-dom";
```

Comment:
What is the usage of `useHistory` here?

Reply:
fixed in e3f02ac73e2598e97bbef38524ad149927667e94